src/Components/Lobby/Lobby.scss

```diff
@@ -0,0 +1,26 @@
+.Lobby {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  text-align: center;
+  justify-content: center;
+  height: 80vh;
+  width: 80vw;
+}
+.moodButton {
+  border-radius: 100px;
+  margin:1vh;
+  width: 15vh;
```

Comment:
Nice way of using vh on both height and width to keep a 1:1 apsect ration on these buttons!